src/symbol.rs

```diff
@@ -0,0 +1,89 @@
+use crate::HandleScope;
+use crate::Isolate;
+use crate::Local;
+use crate::String;
+use crate::Symbol;
+use crate::Value;
+
+extern "C" {
+  fn v8__Symbol__New(
+    isolate: *mut Isolate,
+    description: *const String,
+  ) -> *const Symbol;
+  fn v8__Symbol__ForApi(
+    isolate: *mut Isolate,
+    description: *const String,
+  ) -> *const Symbol;
+  fn v8__Symbol__Description(this: *const Symbol) -> *const Value;
+}
+
+macro_rules! well_known {
+  ($name: ident, $binding: ident) => {
```

Comment:
Idiomatically you'd omit the space between the bound name and AST type, so `$name:ident` rather than `$name: ident`.